beacon-chain/core/blocks/validity_conditions.go

```diff
@@ -79,3 +82,25 @@ func IsSlotValid(slot uint64, genesisTime time.Time) bool {
 	}
 	return isValid
 }
+
+// DepositsToRemove returns the deposits that we have to remove.
+func DepositsToRemove() []*pb.Deposit {
```

Comment:
I dont think these methods should be in `validity_conditions.go`, they feel more block operations to me, or maybe a new file deposit operations

Reply:
sg , will put it in a new file

Reply:
Thank you!